android/src/main/java/com/brentvatne/react/ReactVideoView.java

```diff
@@ -258,7 +258,11 @@ public void setSrc(final String uriString, final String type, final boolean isNe
         mEventEmitter.receiveEvent(getId(), Events.EVENT_LOAD_START.toString(), event);
 
         // not async to prevent random crashes on Android playback from local resource due to race conditions
-        prepare(this);
+        try {
+          prepare(this);
+        } catch (Exception e) {
+          e.printStackTrace();
+        }
```

Comment:
Got a compile error without. didn't happen before has something changed in master?